buzzmobile/launch/simulation_test.launch

```diff
@@ -0,0 +1,30 @@
+<launch>
+    <arg name="mission_control" default="false"/>
+    <arg name="gzclient" default="false" />
+    <arg name="world" default="$(find buzzmobile)/simulation/models/world/straight_road.world"/>
+
+    <group ns="buzzmobile">
+        <include file="$(find buzzmobile)/launch/includes/params.launch"/>
+        <include file="$(find buzzmobile)/launch/includes/gps.launch"/>
+        <!-- <include file="$(find buzzmobile)/launch/includes/mapping.launch"/> -->
+        <include file="$(find buzzmobile)/launch/includes/steering.launch"/>
+        <include file="$(find buzzmobile)/launch/includes/lidar.launch"/>
+        <include file="$(find buzzmobile)/launch/includes/controller.launch"/>
+        <group if="$(arg mission_control)">
+            <include file="$(find buzzmobile)/launch/includes/visualization.launch"/>
+        </group>
+        <include file="$(find buzzmobile)/launch/includes/sim_interface.launch"/>
+    </group>
+
+    <!-- Start gazebo server, load world, optionally start viz client -->
+    <node pkg="gazebo_ros" name="gazebo" type="gzserver"
```

Comment:
I'm getting a bunch of warnings on running this saying "deprecated syntax". can you replicate?

Reply:
I don't see those, no.

Reply:
Oh wait I'm testing these on kinetic, pushing a reminder to myself to test it on indigo in my VM tomorrow.